core/src/layers/prometheus.rs

```diff
@@ -124,29 +133,37 @@ pub struct PrometheusMetrics {
 
 impl PrometheusMetrics {
     /// new with prometheus register.
-    pub fn new(registry: Registry) -> Self {
+    pub fn new(
```

Comment:
This is not our public API, please add those API in `PrometheusLayer`. And I prefer to call them `requests_duration_seconds_buckets` and `bytes_total_buckets` to align with our metrics.

Reply:
Fuck, I'm idiot(